platform/web/detect.py

```diff
@@ -99,7 +102,9 @@ def configure(env: "Environment"):
         env.Append(CPPFLAGS=["-fno-exceptions"])
 
     env.Append(LINKFLAGS=["-s", "INITIAL_MEMORY=%sMB" % env["initial_memory"]])
-
+    # "TOTAL_STACK" was renamed to "STACK_SIZE" in Emscripten 3.1.25, but still works as an alias
```

Comment:
```suggestion
    # "TOTAL_STACK" was renamed to "STACK_SIZE" in Emscripten 3.1.25, but still works as an alias.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide), assuming this comment is necessary